errai-data-binding/src/test/java/org/jboss/errai/databinding/client/scan/TestModelBindable.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2011 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jboss.errai.databinding.client.scan;
+
+import org.jboss.errai.common.client.api.annotations.Portable;
+import org.jboss.errai.databinding.client.api.Bindable;
+import org.junit.Ignore;
+
+/**
+ * Simple bindable model for testing purposes (needs to be configured as
+ * bindable type in ErraiApp.properties).
+ * 
+ * @author Christian Sadilek <csadilek@redhat.com>
```

Comment:
Please remove this author tag (if you'd like you may add an author tag for yourself in its place).